ui/tests/helpers/oidc-config.js

```diff
@@ -116,6 +121,15 @@ export const CLIENT_DATA_RESPONSE = {
   redirect_uris: [],
 };
 
+export const ASSIGNMENT_LIST_RESPONSE = {
+  keys: ['test-assignment'],
```

Comment:
should we include 'allow_all' here as it will always be in the keys array?

Reply:
yeah, might as well. I'll go ahead and add it.